src/micuconversions.cpp

```diff
@@ -326,7 +330,11 @@ QString MIcuConversions::icuDatePatternEscaped(const QString &str)
 QString MIcuConversions::parseOption(const QString &localeName, const QString &option)
 {
     QString value;
+#if QT_VERSION < 0x051500
     QRegExp regexp("^[^@]+@.*"+QRegExp::escape(option)+"=([^@=;]+)($|;.*$)");
+#else
```

Comment:
QRegularExpressions exist since Qt 5.0, unless you using a Qt 5.15 feature they should work earlier.

Reply:
QRegExp was removed